docs/api-docs/storefront/graphql/graphql-api-overview.mdx

```diff
@@ -194,60 +189,74 @@ Content-Type: application/json
 </Tab>
 </Tabs>
 
-Customer impersonation token-authenticated requests made to the GraphQL Storefront API receive store information from the perspective of the customer corresponding to the customer ID specified in the `X-Bc-Customer-Id` header sent with the GraphQL `POST` request. Pricing, product availability, customer account, and customer details will be reflected.
+Customer impersonation token-authenticated requests made to the GraphQL Storefront API may query store information from the perspective of ANY customer by specifying the customer ID using the `X-Bc-Customer-Id` header sent with the GraphQL `POST` request. Customer-specific data such as product pricing & availability, customer account details, and other parameters will change to reflect the specified customer's data.
 
-Customer impersonation tokens should **never** be exposed publicly, for example, to JavaScript or HTML. These tokens should not be used for front-end requests. Unlike normal GraphQL Storefront API tokens, they are sensitive and should be treated like secrets, just as you might treat an OAuth access token for BigCommerce's REST APIs. Attempts to run requests using these tokens from a web browser will be rejected.
+Unlike typical GraphQL Storefront API tokens, Customer impersonation tokens _are_ sensitive, and should therefore **never** be exposed publicly, for example, to JavaScript or HTML. These tokens should not be used for front-end requests. They should be treated with the same care as other application secrets, just as you might treat an OAuth access token for BigCommerce's REST APIs. Requests using these tokens originating from a web browser will be rejected.
 
 Consider this sample request using a Customer Impersonation token to run a request in the context of customer ID `123`.
 
 ```shell filename="Example request: Query as customer 123" showLineNumbers copy
 curl 'https://store.com/graphql' -H 'Authorization: Bearer TOKEN_GOES_HERE' -H 'X-Bc-Customer-Id: 123' --data-binary '{"query":"query CustomerInformation {\n  customer {\n    firstName\n    lastName\n    email\n  }\n}"}'
 ```
 
+<Callout type="info">
```

Comment:
Moving this callout down to the Customer Impersonation token section, as this is the more sensitive token.